src/graph/UpdateEdgeExecutor.cpp

```diff
@@ -249,6 +249,26 @@ void UpdateEdgeExecutor::execute() {
             return;
         }
         auto rpcResp = std::move(resp).value();
+        for (auto& code : rpcResp.get_result().get_failed_codes()) {
```

Comment:
A little quetion, is there return many failed codes? if it is, we just process the first failed code in the follow logic, because we return at each case.

Reply:
Normally there will only be one code in failed_codes, because the current update operation only happens on one partition. Here only the first code is checked and the prompt is returned.